activitysim/abm/models/tour_mode_choice.py

```diff
@@ -123,6 +135,61 @@ def tour_mode_choice_simulate(tours, persons_merged,
     primary_tours_merged['tour_purpose'] = \
         primary_tours_merged.tour_type.where(not_university, 'univ')
 
+    # if trip logsums are used, run trip mode choice
+    if model_settings.get('COMPUTE_TRIP_MODE_CHOICE_LOGSUMS', False):
+
+        # Construct table of hypothetical trips from tours for each potential
+        # tour mode. Two trips (1 inbound, 1 outbound) per [tour, mode] bundle.
+        # O/D, purpose, and departure times are inherited from tour.
+        primary_tours_merged['stop_frequency'] = '0out_0in'  # no intermediate stops
+        primary_tours_merged['primary_purpose'] = primary_tours_merged['tour_purpose']
+        trips = trip.initialize_from_tours(primary_tours_merged)
+        trips['stop_frequency'] = '0out_0in'
```

Comment:
If '0out_0in' is to be used, it should be set in a variable. Effectively, this is a "magic number"

Reply:
If I'm understanding you correctly, this is fixed.